### ISSUE TITLE
Doesn't start on Windows/Linux

### ISSUE THREAD

**Issue by nilsreiter** [nilsreiter/CorefAnnotator#68]
```
Exception in thread "AWT-EventQueue-0" java.lang.ArrayIndexOutOfBoundsException: No such child: 4
       at java.desktop/java.awt.Container.getComponent(Unknown Source)
       at de.unistuttgart.ims.coref.annotator.comp.XFileChooser.<init>(XFileChooser.java:24)
       at de.unistuttgart.ims.coref.annotator.Annotator.initialiseDialogs(Annotator.java:139)
       at de.unistuttgart.ims.coref.annotator.Annotator.<init>(Annotator.java:134)
       at de.unistuttgart.ims.coref.annotator.Annotator$1.run(Annotator.java:107)
       at java.desktop/java.awt.event.InvocationEvent.dispatch(Unknown Source)
       at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
       at java.desktop/java.awt.EventQueue.access$500(Unknown Source)
       at java.desktop/java.awt.EventQueue$3.run(Unknown Source)
       at java.desktop/java.awt.EventQueue$3.run(Unknown Source)
       at java.base/java.security.AccessController.doPrivileged(Native Method)
       at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
       at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
       at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
       at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
       at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
       at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
       at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
       at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
```

This is caused by the file open dialog looking differently on different UIs. Stupid me, should have thought of that.

**Comment by nilsreiter** [comment]
There is probably no cross-platform way to do that.

**Comment by nilsreiter** [comment]
Fixed in 2f9797a.
